apps/cowswap-frontend/src/modules/tokensList/pure/ModalHeader/index.tsx

```diff
@@ -11,31 +11,45 @@ const Header = styled.div`
   justify-content: space-between;
   font-weight: 500;
   font-size: 20px;
-  padding: 20px;
-  border-bottom: 1px solid var(${UI.COLOR_BORDER});
+  padding: 10px 0;
+
+  ${IconButton} {
```

Comment:
Please, avoid using this approach.
It's better to use:
 - `const StyledIconButton = styled(IconButton)`
 - or props: `<IconButton type="myType"/>`

Reply:
When we use `${IconButton} {` we break Dependency Inversion principle, it may cause circular dependencies and bundle size problems

Reply:
There is an example how to refactor issues like that:
https://github.com/cowprotocol/cowswap/pull/3227/commits/0abd19e3c940e63d1596fd580279d41436dca982

Reply:
got it! will refactor